src/get-user-code-frame.ts

```diff
@@ -29,12 +33,14 @@ function getCodeFrame(frame) {
 
   let rawFileContents = ''
   try {
-    rawFileContents = readFileSync(filename, 'utf-8')
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    rawFileContents = readFileSync?.(filename, 'utf-8')
```

Comment:
This would no longer throw if `readFileSync` is not callable. 

Generally, don't change any runtime when porting to TypeScript. Use type casting instead so that we know where we might want to work on.